src/cosalib/aws.py

```diff
@@ -50,7 +51,8 @@ def aws_run_ore_replicate(build, args):
 
     region_list = list(set(args.region) - set(duplicates))
     if len(region_list) == 0:
-        raise Exception("no new regions detected")
```

Comment:
fwiw, the original reason to raise the exception was it to be evaluated by the `cmd-ore-wrapper` command.  But since it was decided to not pursue Pythonic libraries here, that work was never pursued (and why the exception is a generic `Exception` instead of a distinct one.